writers_rollingfilewriter.go

```diff
@@ -698,28 +710,12 @@ func (rwt *rollingFileWriterTime) sortFileRollNamesAsc(fs []string) ([]string, e
 	return ss.data, nil
 }
 
-func (rwt *rollingFileWriterTime) getNewHistoryRollFileName(lastRollName string) string {
-	return ""
+func (rwt *rollingFileWriterTime) getNewHistoryRollFileName(lastRoll rollInfo) string {
+	return lastRoll.Time.Format(rwt.timePattern)
```

Comment:
RollFileName is the prefix/suffix for the rolled file. I had a similar reaction and tried to jam the `originalFileName` into this. That's how I ended up with `axiom-console.log.axiom-console.log-2016-05-19`